src/cpp/r/RUtil.cpp

```diff
@@ -147,11 +147,7 @@ core::Error iconvstr(const std::string& value,
                      std::string* pResult)
 {
    std::string effectiveFrom = from;
-   if (effectiveFrom.empty())
```

Comment:
Even if we believe this change needs to be made, it dramatically increases the surface area of this patch since we call `iconvstr` all over the place. As this is a last-minute change before shipping, let's limit the changes to the find and replace codepath so that we can feel 100% confident we haven't broken or changed any other behavior.

Reply:
That's a good point. Change has been reverted; I still removed the lines where I was setting empty to "UTF-8" in `SessionFind` as it doesn't change any logic and is inaccurate.

Reply:
👍